app/api/v1/endpoints/admin.py

```
@@ -2961,4 +2976,48 @@ async def review_strategy(
         raise HTTPException(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
             detail="Failed to review strategy"
-        ) from e
\ No newline at end of file
+        ) from e
+
+@router.get("/signals/deliveries", response_model=List[SignalDeliveryAudit])
+async def admin_signal_deliveries(
+    limit: int = Query(100, ge=1, le=500),
```

Ruff (0.13.3)

Undefined name `Query`

(F821)